travis.sh

```diff
@@ -191,19 +191,17 @@ BUILD)
         -Dsource.skip=true \
         -Pdeploy-sonarsource,release
 
-    # TODO remove the sonar.pullrequest.github.* settings after sonar-core-plugins 7.1.0.330 is deployed on Next
     mvn sonar:sonar \
         -Dsonar.host.url=$SONAR_HOST_URL \
         -Dsonar.login=$SONAR_TOKEN \
-        -Dsonar.branch.name=$TRAVIS_PULL_REQUEST_BRANCH \
-        -Dsonar.branch.target=$TRAVIS_BRANCH \
         -Dsonar.analysis.buildNumber=$BUILD_NUMBER \
         -Dsonar.analysis.pipeline=$BUILD_NUMBER \
         -Dsonar.analysis.sha1=$TRAVIS_PULL_REQUEST_SHA \
         -Dsonar.analysis.prNumber=$TRAVIS_PULL_REQUEST \
         -Dsonar.analysis.repository=$TRAVIS_REPO_SLUG \
-        -Dsonar.pullrequest.id=$TRAVIS_PULL_REQUEST \
-        -Dsonar.pullrequest.github.id=$TRAVIS_PULL_REQUEST \
+        -Dsonar.pullrequest.key=$TRAVIS_PULL_REQUEST \
+        -Dsonar.pullrequest.branch=$TRAVIS_PULL_REQUEST_BRANCH \
+        -Dsonar.pullrequest.base=$TRAVIS_BRANCH \
```

Comment:
@teryk You need the provider no?

Reply:
we have set token, repository and provider automatically for all SonarSource projects, but indeed in Travis the provider will always be “gihub”. I'll add it